map/map_loader/src/lanelet2_map_loader/lanelet2_map_loader_node.cpp

```diff
@@ -93,6 +100,21 @@ lanelet::LaneletMapPtr Lanelet2MapLoaderNode::load_map(
     if (errors.empty()) {
       return map;
     }
+  } else if (lanelet2_map_projector_type == "local") {
+    // Use MGRSProjector as parser
+    lanelet::projection::MGRSProjector projector{};
+    const lanelet::LaneletMapPtr map = lanelet::load(lanelet2_filename, projector, &errors);
+
+    // overwrite local_x, local_y
+    for (lanelet::Point3d point : map->pointLayer) {
+      if (point.hasAttribute("local_x")) {
+        point.x() = point.attribute("local_x").asDouble().value();
+      }
+      if (point.hasAttribute("local_y")) {
+        point.y() = point.attribute("local_y").asDouble().value();
+      }
+    }
+    return map;
```

Comment:
I think this should be created as a different PR since adding "local" option to the loader and adding a feature to publish map projector info is different.

Reply:
@mitsudome-r 
The code is reverted and move to https://github.com/autowarefoundation/autoware.universe/pull/3492